nucypher/blockchain/eth/registry.py

```diff
@@ -184,7 +187,7 @@ def search(self, contract_name: str = None, contract_address: str = None) -> tup
             raise self.UnknownContract(contract_name)
 
         if contract_address and len(contracts) > 1:
-            m = "Multiple records returned for address {}"
+            m = f"Multiple records returned for address {contract_address}"
             self.log.critical(m)
             raise self.IllegalRegistry(m.format(contract_address))
```

Comment:
You mean, `raise self.IllegalRegistry(m)`?

Reply:
✅